sdks/python/container/py313/base_image_requirements.txt

```diff
@@ -0,0 +1,171 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# Autogenerated requirements file for Apache Beam py313 container image.
+# Run ./gradlew :sdks:python:container:generatePythonRequirementsAll to update.
+# Do not edit manually, adjust ../base_image_requirements_manual.txt or
+# Apache Beam's setup.py instead, and regenerate the list.
+# You will need Python interpreters for all versions supported by Beam, see:
+# https://s.apache.org/beam-python-dev-wiki
+# Reach out to a committer if you need help.
+
+annotated-types==0.7.0
+anyio==4.9.0
+attrs==25.3.0
+beautifulsoup4==4.13.4
+bs4==0.0.2
+build==1.2.2.post1
+cachetools==5.5.2
+certifi==2025.4.26
+cffi==1.17.1
+charset-normalizer==3.4.2
+click==8.2.0
+crcmod==1.7
+cryptography==45.0.2
+Cython==3.1.1
+Deprecated==1.2.18
+deprecation==2.1.0
+dill==0.3.1.1
+dnspython==2.7.0
+docker==7.1.0
+docopt==0.6.2
+docstring_parser==0.16
+execnet==2.1.1
+fastavro==1.11.1
+fasteners==0.19
+freezegun==1.5.1
+future==1.0.0
+google-api-core==2.24.2
+google-apitools==0.5.32
+google-auth==2.40.1
+google-auth-httplib2==0.2.0
+google-cloud-aiplatform==1.93.1
+google-cloud-bigquery==3.33.0
+google-cloud-bigquery-storage==2.31.0
+google-cloud-bigtable==2.30.1
+google-cloud-core==2.4.3
+google-cloud-datastore==2.21.0
+google-cloud-dlp==3.29.0
+google-cloud-language==2.17.1
+google-cloud-pubsub==2.29.0
+google-cloud-pubsublite==1.12.0
+google-cloud-recommendations-ai==0.10.17
+google-cloud-resource-manager==1.14.2
+google-cloud-spanner==3.54.0
+google-cloud-storage==2.19.0
+google-cloud-videointelligence==2.16.1
+google-cloud-vision==3.10.1
+google-crc32c==1.7.1
+google-genai==1.16.1
+google-resumable-media==2.7.2
+googleapis-common-protos==1.70.0
+greenlet==3.2.2
+grpc-google-iam-v1==0.14.2
+grpc-interceptor==0.15.4
+grpcio==1.72.0rc1
```

Comment:
was this manually edited to install an RC version? does it have the fix?

Reply:
that was not intended, I generated this file with the generatePythonRequirements workflow and it seems to pick up RC versions when I don't want it to. I bumped it back down to 1.71.0